src/main/java/blackjack/domain/Cards.java

```diff
@@ -0,0 +1,44 @@
+package blackjack.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cards {
+
+    private final List<Card> cards;
+
+    public Cards(List<Card> cards) {
+        this.cards = cards;
+    }
+
+    public void addCard(Card card) {
+        cards.add(card);
+    }
+
+    public int getScore() {
+        int total = cards.stream().mapToInt(card -> card.getDenomination().getScore()).sum();
```

Comment:
`card.getScore()`라는 메서드를 만들어보면 어떨까요?

```suggestion
        int total = cards.stream().mapToInt(card -> card.getScore()).sum();
```